corehq/tests/nose.py

```diff
@@ -171,27 +172,22 @@ class HqdbContext(DatabaseContext):
 
     Other supported `REUSE_DB` values:
 
-    - `REUSE_DB=reset` : drop existing, then create and migrate new test
-      databases, but do not teardown after running tests. This is
-      convenient when the existing databases are outdated and need to be
-      rebuilt.
-    - `REUSE_DB=teardown` : skip database setup; do normal teardown after
-      running tests.
-    - `REUSE_DB=flush` : same as `REUSE_DB=1` except delete all objects from the
-      databases before running tests.  Much faster than `reset`.
-    - `REUSE_DB=migrate` : same as `REUSE_DB=1` except migrate databases
-      before running tests.
+    The following flags are also useful in conjunction with REUSE_DB=1:
+        --reset-db
+        --flush-db
+        --migrate-db
+        --teardown-db
     """
 
     def __init__(self, tests, runner):
-        self.reuse_db = reuse_db = os.environ.get("REUSE_DB")
-        self.skip_setup_for_reuse_db = reuse_db and reuse_db != "reset"
-        self.skip_teardown_for_reuse_db = reuse_db and reuse_db != "teardown"
+        reuse_db = CmdLineParametersPlugin.get('db_action') or os.environ.get("REUSE_DB")
```

Comment:
Looks like `REUSE_DB=flush_db` would also work. Is that intentional? I think you want `bool(os.environ.get("REUSE_DB"))`